### DISCUSSION TITLE
WEBBOX-G2

### DISCUSSION THREAD

**Discussion by chma73** [jpcornil-git/HA-sma#11] [Q&A] [answered]
Hello,
do you know if it also works with the WEBBOX-G2,  Firmware 1.53 and RS485 communication?
Thanks for supporting.

**Comment by jpcornil-git** [comment, marked as answer]
Hi,
I don't have one/can't say for sure but I guess that RS485 is for the connection to the PV inverter (iso bluetooth for mine) and that you have a network connection. If this is the case, download [sma_webbox.py](https://github.com/jpcornil-git/HA-sma/blob/main/custom_components/sma_webbox/sma_webbox.py) and run
```bash
python3 sma_webbox.py <ip address of your webbox>
```
If you you see raw data coming out of the script, it should work.

Cheers,
jpc

**Reply by chma73** [reply]
Hello jpc,
yes the RS485 is the connection to the SMA inverter and the WebBox itself is connected with lan to the network.
I tried it and it works perfect :-)

**Comment by jpcornil-git** [comment]
Glad to hear this and thanks to report back :-)
